spec/models/spree/variant_spec.rb

```diff
@@ -60,14 +60,12 @@
 
   context "price parsing" do
     before(:each) do
-      I18n.locale = I18n.default_locale
-      I18n.backend.store_translations(:de,
-                                      { number: { currency: { format: { delimiter: '.',
-                                                                        separator: ',' } } } })
-    end
-
-    after do
-      I18n.locale = I18n.default_locale
+      default_locale = I18n.default_locale
+      I18n.with_locale(default_locale) do
+        I18n.backend.store_translations(:de,
+                                        { number: { currency: { format: { delimiter: '.',
+                                                                          separator: ',' } } } })
+      end
```

Comment:
I think this would be better as an `around` , otherwise the examples below won't run with `default_locale` applied.  
```suggestion
  around(:each) do |spec|
     I18n.with_locale(default_local) do
       I18n.backend.store_translations(:de,
                                        { number: { currency: { format: { delimiter: '.',
                                                                          separator: ',' } } } })
        spec.run
      end
  do
```

Reply:
Oh...My bad! 😅
Thanks for pointing that out ⭐